src/fix.rs

```diff
@@ -0,0 +1,77 @@
+use diesel;
+use diesel::prelude::*;
+use diesel::types::{Integer, VarChar};
+
+use github::{update_issue, update_pr};
+
+use DB_POOL;
+
+pub fn fix() {
+    for (repo, number) in get_duplicate_issues() {
+        delete_duplicate_issues(&repo, number);
+        update_issue(&repo, number).expect("Failed to update issue");
```

Comment:
Hm... so in my ideal world, we would just have the `delete, insert` operations wrapped in a single transaction (one transaction per duplicate issue/PR). Each issue/PR is independent of the others, so we can just re-run the script if fails after only processing a couple issues.

That said, I can't see how to wrap it in a transaction without refactoring a _bunch_ of code. Transactions in diesel need to share a common `PgConnection`, which means we'd need to update `update_issue` and `update_pr` to take `conn` as a parameter instead of them calling `DB_POOL.get()`. (Although, this might be a good idea if we want to add tests).

Do you think that's a good idea, or should I stick with reordering?